src/components/manage/s3/LrS3Access.vue

```diff
@@ -113,13 +117,17 @@ export default class LrS3Access extends Vue {
   newAccessKey = {};
   generatedAccessKeyDialog = false;
   deleteModal = create<SgtDialogModel>(SgtDialog);
+  status = "active";
 
   mounted() {
     this.getAccessKeys();
   }
   getAccessKeys() {
     Api.getData("s3/access_keys", { isDummy: true }).then((resp: any) => {
       this.accessList = resp["access_keys"];
+      this.accessList = resp["access_keys"].map((ele: any) => {
+        return { ...ele, statusFlag: ele.status == "active" ? true : false };
```

Comment:
Would suggest using strict equality check '===' and you could also remove the ternary operator for returning boolean values as the equality check operator itself returns a boolean value.

Reply:
done